samples/client/petstore/go/go-petstore/model_name.go

```diff
@@ -165,15 +165,11 @@ func (o Name) MarshalJSON() ([]byte, error) {
 	if true {
 		toSerialize["name"] = o.Name
 	}
-	if !isNil(o.SnakeCase) {
-		toSerialize["snake_case"] = o.SnakeCase
-	}
+    // skip: snake_case is readOnly
 	if !isNil(o.Property) {
 		toSerialize["property"] = o.Property
 	}
-	if !isNil(o.Var123Number) {
-		toSerialize["123Number"] = o.Var123Number
-	}
+    // skip: 123Number is readOnly
 	return json.Marshal(toSerialize)
 }
 
```

Comment:
can we add a test with `model_name` to ensure readonly fields are not serialized?

Reply:
@wing328 I have added already a test in `GoClientCodegenTest`, but I think you are asking to add a different test. Can you point me to the right direction (what/where to change)? Tnx